spec/system/page_objects/components/legacy_header_dropdown.rb

```diff
@@ -8,7 +8,7 @@ def click
       end
 
       def visible?
-        page.has_css?(".menu-panel.drop-down")
+        page.has_css?(".menu-container-general-links")
```

Comment:
This resulted in a false positive previously because both the legacy and sidebar header dropdown has the `.menu-panel.drop-down` CSS class. The only difference is that the sidebar dropdown has an extra `.revamped` class.